packages/headless/src/modules/wishlist/graphql/wishlist.controller.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Controller, Get, Param, Res } from '@nestjs/common';
+import { Response } from 'express';
+import { WishListService } from '../services';
+
+@Controller('wishlist')
```

Comment:
If we don't need this controller in graphql then make the file empty. Don't keep unused code in commit